test/primer/hyperloglog_test.cpp

```diff
@@ -22,7 +21,7 @@ TEST(HyperLogLogTestHyperLogLogTest, DISABLED_BasicTest1) {
   obj.ComputeCardinality();
   auto ans = obj.GetCardinality();
 
-  ASSERT_EQ(ans, 0);
+  ASSERT_EQ(ans, 2);
```

Comment:
Maybe I've implemented this incorrectly at the moment, but I keep getting 3 here for classic HLL(not Presto). Is this changes are final?

Reply:
Using std::floor?

Reply:
Yeah, for the first case, I'm retrieving 3.07984. When casted to size_t -> 3.
I can share my calculations if you need it

Reply:
Don't post anything here. Can lead to AIV. I think it may differ with the way you are calculating cardinality. 
Will post in website soon.
values should be final.

Reply:
Sure! Thanks, gonna wait

Reply:
@prashanthduvvada Hey! I figured out the problem, and I suggest you include that info on the course page. std::hash works differently on different CPU architectures (on ARM on x86 for sure), so that can cause issues for guys who going to try to implement primer on ARM MacBooks.